### ISSUE TITLE
[Feature request] Refresh/reload text

### ISSUE THREAD

**Issue by ramontayag** [jaz303/tipsy#33]
It'd be great if you could call refresh or reload on tipsy so it could refer back to the trigger element it's linked to, read the title (or whatever it's linked to), and update the text on the tooltip.

**Comment by jaz303** [comment]
Tipsy already reloads its content every time it's triggered. Or did you mean reload it while it's still visible?

**Comment by ramontayag** [comment]
Yup, that's what I mean. Much like github's, when you click on the clipboard the text changes to "copied!". :)

**Comment by jaz303** [comment]
OK i'll look into this.

**Comment by pokonski** [comment]
Any plans on implementing? I might look into coding this feature.

**Comment by bkw** [comment]
updating the title and calling tipsy('show') again works for me.
